pkg/osvscanner/vulnerability_result.go

```diff
@@ -4,6 +4,7 @@ import (
 	"sort"
 	"strings"
 
+	out "github.com/google/osv-scanner/internal/output"
```

Comment:
happy for this to be done in a follow up pr sometime, but ideally it would be better to rename the variable